.gitmodules

```
@@ -14,5 +14,5 @@
 	path = FreeRTOS-Plus/Source/Application-Protocols/coreMQTT
 	url = https://github.com/FreeRTOS/coreMQTT
 [submodule "FreeRTOS-Plus/Source/Application-Protocols/coreJSON"]
```

Does this not need to be changed as well?
```suggestion
[submodule "FreeRTOS-Plus/Source/coreJSON"]
```